### DISCUSSION TITLE
Organize stars with individual labels

### DISCUSSION THREAD

**Discussion by fynnfluegge** [community/community#52461] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi 👋,
I enjoy the list feature for my stars. 
However, I would love the feature to tag starred projects with individual labels similar to issues in a repo. I think it would make the structure of starred projects even cleaner. It's common to have over hundred or even hundreds of stars. Filtering all stars by a label would be more handy to me for fast access of a topic of starred projects. And filtering by multiple labels will make it possible to search for topic intersections which is currently not possible with lists.

**Comment by Robleigh23o** [comment]
[](){@}

**Reply by Robleigh23o** [reply]
BY THE SEAT OV MY BRICHES
